<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editNoteColorSchemeDlg</class>
 <widget class="QDialog" name="editNoteColorSchemeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Scheme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ColorButton" name="textColorButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="postnote.qrc">
         <normaloff>:/PostNote/Resources/Text Foreground.png</normaloff>:/PostNote/Resources/Text Foreground.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorButton" name="bgColorButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="postnote.qrc">
         <normaloff>:/PostNote/Resources/Text Background.png</normaloff>:/PostNote/Resources/Text Background.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="sampleLabel">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>Resulting Note Appearance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Transparency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="transparencySlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="transparencyValueLabel">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>color_button</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="postnote.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editNoteColorSchemeDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editNoteColorSchemeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
